ansible/README.md

```diff
@@ -113,7 +117,7 @@ root of this directory structure. It should contain key/value pairs of variable
 names and credentials to be used in the playbooks. Currently, the only required
 credentials are quay_username and quay_password, which are created by make
 from the environment variables `QUAY_RHACS_ENG_RO_USERNAME` and `QUAY_RHACS_ENG_RO_PASSWORD`
-to match CI variables.
+to match CI variables. If you are using IBM Cloud to create IBM Z RHEL instances, you will also need to specifiy enviroment variables `RHEL_SUBSCRIPTION_USERNAME` and `RHEL_SUBSCRIPTION_PASSWORD` to register your RHEL system.
```

Comment:
I believe we can use `REDHAT_USERNAME` and `REDHAT_PASSWORD`, our CI has those setup to a RHSM account for the team.
```suggestion
to match CI variables. If you are using IBM Cloud to create IBM Z RHEL instances, you will also need to specifiy enviroment variables `REDHAT_USERNAME` and `REDHAT_PASSWORD` to register your RHEL system.
```